c/datatable.h

```diff
@@ -83,6 +95,15 @@ class DataTable {
      */
     RowIndex sortby(const arr32_t& colindices, Groupby* out_grps) const;
 
+    const std::vector<std::string>& get_names() const;
+    py::otuple get_pynames() const;
+    int64_t colindex(const py::_obj& pyname) const;
+    void copy_names_from(const DataTable* other);
+    void set_names_to_default();
+    void set_names(const py::olist& names_list);
+    void set_names(const std::vector<std::string>& names_list);
+    void replace_names(py::odict replacements);
```

Comment:
So to replace particular names the only option is to use `py::odict`?

Reply:
Yes, because we have no other use cases for now. If in the future a use case arises to replace, say, from an `std::unordered_map`, we can add that function then.